### DISCUSSION TITLE
Bizarre error from Copilot

### DISCUSSION THREAD

**Discussion by PythonCoderAS** [community/community#41141] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I opened a new window on VSCode and got this error:

```
[ERROR] [default] [2022-12-08T22:10:07.590Z] GitHub Copilot could not connect to server. Extension activation failed: "child "activity" fails because [child "buttons" fails because ["buttons" at position 0 fails because [child "url" fails because ["url" must be a valid uri]]]]"
```

I am not sure what this is supposed to mean.

**Comment by iamcjwilliams** [comment, marked as answer]
@PythonCoderAS Sorry that you're facing trouble using Copilot with VSCode. It seems like there is a problem with connecting to the Copilot server. Please uninstall and then reinstall the plugin, allow it some time to connect, and see if that helps. Also, make sure that you are running the latest version of both Copilot and VSCode

**Reply by PythonCoderAS** [reply]
It fixed itself after a restart of VSCode.

**Comment by mcitir** [comment]
Hello, I have a same error. Unfortunately, I restart, and reinstall, nothing happened to fix the error.
